src/app/components/team-page/team-page.component.html

```diff
@@ -1 +1,47 @@
-<p>team-page works!</p>
+<div class="container">
+  <div id="speakers">
+    <div class='row center-lg center-xs center-md'>
+      <div class="col-xs-12">
+        <app-homepage-title title='Team' color='#419BF9'></app-homepage-title>
+      </div>
+      <p>DevFest Gandhinagar is brought to you with 💓 love,
```

Comment:
Can you replace this emoji with the simple :heart: emoji

Reply:
I tried but couldn't because instead of red heart it displays white heart.

Reply:
@aashutoshrathi modified as per required please review again.